streamparse/util.py

```diff
@@ -259,6 +259,31 @@ def get_nimbus_client(env_config=None, host=None, port=None, timeout=7000):
                                 timeout=timeout)
     return nimbus_client
 
+_storm_workers = {}
+def get_storm_workers(env_config):
+    """Retrieves list of workers, optionally from nimbus
+
+    This function will look up the list of current workers from nimbus if
+    workers has not been defined in config.json
+    :param env_config: The project's parsed config.
+    :type env_config: `dict`
+
+    :returns: List of workers
+    """
+    nimbus_info = get_nimbus_host_port(env_config)
+    if nimbus_info in _storm_workers:
+        return _storm_workers[nimbus_info]
+
+    worker_list = env_config.get('workers', None)
```

Comment:
Nitpick: the `, None` here is unnecessary. `dict.get` returns `None` by default if the key isn't present.